lib/server/forwardClientRequestToApi.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var request = require("request-promise");
```

Comment:
instead of introducing a new library, you can use bluebird like this:

```js
var Promise = require("bluebird");
var requestPromise = Promise.promisify(require("request"));
```

you might have to use `spread` instead of `then`:

```js
request(options)
    .spread(httpResponse, responseBody) {
        var statusCode = httpResponse.statusCode;
      
        res.status(statusCode).send(responseBody);
    });
```